.github/workflows/update-nibble.yml

```diff
@@ -0,0 +1,33 @@
+name: Update Nibble Link
+
+on:
+  schedule:
+    # Schedule to run at 6 PM IST every Thursday
+    # 12:30 PM UTC is 6 PM IST
+    - cron: '30 12 * * 4'
+
+jobs:
+  update-link:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: '14'
+
+      - name: Update Nibble Link
+        run: node update-nibble.js
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          commit-message: "Increment Nibble link"
+          title: "Weekly Automated Nibble Link Update"
+          body: "This is an automated pull request to increment the Nibble link."
+          branch: "update-nibble-link"
+          delete-branch: true
+          author: "Nibble Bot <nibblebot@nibbles.dev/>"
```

Comment:
Lol what's this auto-close of mail tag 🤣

Reply:
Fixed, thanks 😂